service/templates/cloudformation/host/main.yaml

```diff
@@ -0,0 +1,6 @@
+{{define "main"}}AWSTemplateFormatVersion: 2010-09-09
+Description: Main Host CloudFormation stack.
+Resources:
+  {{template "iam_roles" .}}
+  {{template "route_tables" .}}
```

Comment:
I really like the template structure and good to see it in the host stack too! <3